Fomichev/HashingApp/HashingApp/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace HashingApp
+{
+    public static class Program
+    {
+        /// <summary>
+        /// The main entry point for the application.
+        /// </summary>
+        [STAThread]
+        public static void Main()
```

Comment:
Important notice:
Your program can't be run in pure cmdline mode, without creating windows. Actually, the GUI was expected to be just an addition to the cmdline mode.

Reply:
I thought I should replace cmd with GUI. Some misunderstanding happened.